testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/elytron/seccontext/DynamicSSLContextTestCase.java

```diff
@@ -0,0 +1,340 @@
+    private static final String[] CLIENT_SSL_CONTEXT1 = {"subsystem", "elytron", "client-ssl-context", "client1-ssl-context"};
+    private static final String[] CLIENT_SSL_CONTEXT2 = {"subsystem", "elytron", "client-ssl-context", "client2-ssl-context"};
+
+    private static final String[] CLIENT_TRUST_STORE1 = {"subsystem", "elytron", "key-store", "clientTS1"};
+    private static final String[] CLIENT_TRUST_STORE2 = {"subsystem", "elytron", "key-store", "clientTS2"};
+    private static final String[] CLIENT_TRUST_MANAGER1 = {"subsystem", "elytron", "trust-manager", "clientTM1"};
+    private static final String[] CLIENT_TRUST_MANAGER2 = {"subsystem", "elytron", "trust-manager", "clientTM2"};
+
+    private static final String[] AUTHENTICATION_CONTEXT = {"subsystem", "elytron", "authentication-context", "ac"};
+    private static final String[] DYNAMIC_SSL_CONTEXT = {"subsystem", "elytron", "dynamic-client-ssl-context", "dynamicClientSSLContext"};
+    private static final String[] ELYTRON_SUBSYSTEM = {"subsystem", "elytron"};
+
+    private static final String[] HTTP_LISTENER_1 = {"subsystem", "undertow", "server", "default-server", "https-listener", "first-listener"};
+    private static final String[] HTTP_LISTENER_2 = {"subsystem", "undertow", "server=default-server", "https-listener", "second-listener"};
+
+    private static final String[] SOCKET_BINDING_1 = {"socket-binding-group", "standard-sockets", "socket-binding", "first-socket-binding"};
+    private static final String[] SOCKET_BINDING_2 = {"socket-binding-group", "standard-sockets", "socket-binding", "second-socket-binding"};
+
+    private static final String CLIENT_KEYSTORE1_FILENAME = "client.keystore";
+    private static final String CLIENT_TRUSTSTORE1_FILENAME = "client.truststore";
+    private static final String SERVER_KEYSTORE1_FILENAME = "server.keystore";
+    private static final String SERVER_TRUSTSTORE1_FILENAME = "server.truststore";
+    private static final String CONTAINER = "default-jbossas";
+    private static final String DEPLOYMENT = "deployment";
+    private static final String SERVER_ADDRESS = TestSuiteEnvironment.getServerAddress();
+    private static final String JKS = "JKS";
+
+    private static final int PORT1 = 10443;
+    private static final int PORT2 = 11443;
+
+    private static final File KEYSTORES_DIR = new File("target/keystores");
+    private static final File KEYSTORES2_DIR = new File("target/keystores2");
+
+    @ArquillianResource
+    private static ContainerController serverController;
+
+    @ArquillianResource
+    private static Deployer deployer;
+
+    @BeforeClass
+    public static void generateKeystores() throws Exception {
+        if (!KEYSTORES_DIR.exists()) {
+            KEYSTORES_DIR.mkdirs();
+        }
+        if (!KEYSTORES2_DIR.exists()) {
+            KEYSTORES2_DIR.mkdirs();
+        }
+        CoreUtils.createKeyMaterial(KEYSTORES_DIR, JKS);
+        CoreUtils.createKeyMaterial(KEYSTORES2_DIR, JKS);
```

Comment:
The generated keystores/truststores should be removed in an `@AfterClass` method.

Reply:
Fixed